docs/sql-migration-guide.md

```diff
@@ -70,11 +70,11 @@ license: |
 
   - Since Spark 3.2, all the supported JDBC dialects use StringType for ROWID. In Spark 3.1 or earlier, Oracle dialect uses StringType and the other dialects use LongType.
 
-  - In Spark 3.2, PostgreSQL JDBC dialect uses StringType for MONEY and MONEY[] is not supported due to the JDBC driver for PostgreSQL can't handle those types properly. In Spark 3.1 or earlier, DoubleType and ArrayType of DoubleType are used respectively.
+  - Since Spark 3.2, PostgreSQL JDBC dialect uses StringType for MONEY and MONEY[] is not supported due to the JDBC driver for PostgreSQL can't handle those types properly. In Spark 3.1 or earlier, DoubleType and ArrayType of DoubleType are used respectively.
```

Comment:
These still seem like noise changes for no real benefit

Reply:
> These still seem like noise changes for no real benefit

Hmm， OK，  how about just change the error places I mentioned?

Reply:
Right, I think "Since" isn't important to change

Reply:
> Right, I think "Since" isn't important to change

How about current